apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.tsx

```diff
@@ -106,6 +111,13 @@ const AccessSection = (props: Props) => {
         type: actions.UPDATE_MS_ACCOUNT_INFO,
         payload: msAccountInfo,
       });
+
+      trackEvent('configSaved', {
+        action: ConfigAction.Installed,
+        metaData: {
+          tenantId,
+        },
+      });
```

Comment:
hey @rusticpenguin ! just seeing some type errors here, that this method needs an `environment_key`, `app_key` etc. Happy to help adjust these, but want to confirm with you the details here!